tests/Doctrine/Tests/DBAL/Schema/SchemaTest.php

```diff
@@ -449,4 +451,48 @@ public function testVisitsNamespaceVisitor()
 
         $this->assertNull($schema->visit($visitor));
     }
+
+    /**
+     * https://github.com/doctrine/dbal/issues/2538
+     */
+    public function testFixGithub2538Collation()
```

Comment:
Please use a method name describing the issue being covered. Also, use `@group #2538` for labeling it